build.sh

```diff
@@ -111,6 +111,9 @@ else
 fi
 format_error_files=""
 
+# Ignore those PEP8 rules
+PEP8_IGNORES="E121,E123,E126,E226,E24,E704"
```

Comment:
Hi @apeyser,

Without it TravisCI complains, and my local machine not: `extras/parse_travis_log.py:285:45: E126 continuation line over-indented for hanging indent`
The line looks much nicer with this over-indention. On my Mac when I execute `pep8 --help` I get:
```
pep8 --help
Usage: pep8 [options] input ...

Options:
...
  --ignore=errors      skip errors and warnings (e.g. E4,W) (default:
                       E121,E123,E126,E226,E24,E704)
  --show-source        show source code for each error
```
i.e. `E126` is ignored here by default. Maybe we should only ignore `E126`? Also see [here](https://github.com/PyCQA/pycodestyle/issues/167) for discussion.